IndomitableIbises/Bangazon-Sprint-3
# Si order

## Link to Ticket
Closes # [Link]()

## Description of Proposed Changes

- view active order
-view past orders
-view past order details
-complete order
-thank you screen


## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout SI_order_model
```
with no products, orders or payment -
/order will show a blank screen
order history will say there is no order history

with completed orders -
/order_history will display past orders and allow redirect to order detail of any order
/order_detail will show the details of a past order on redirect


with open order and products but no payment -
/order will display, products will display if in shopping cart many-to-many table
no payment will appear and no submit button to allow advance to complete scree

with open order, products and a payment linked to the active user -
/order will show payment options and allow you to complete the order and move to a thank you screen with a redirect to home.



## Impacted Areas in Application

List general components of the application that this PR will affect:

url.py
views init.py
templates folder and views will have additional files


## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
